heron/stmgr/src/cpp/manager/stmgr-client.cpp

```diff
@@ -148,9 +148,6 @@ void StMgrClient::HandleHelloResponse(void*, proto::stmgr::StrMgrHelloResponse*
     Stop();
   }
   delete _response;
-  if (client_manager_->DidAnnounceBackPressure()) {
```

Comment:
it failed the unit test test_back_pressure_stmgr_reconnect.
when the stmgr X in backpressure disconnects and reconnects, stmgr X is supposed to receive backpressure notice according to test_back_pressure_stmgr_reconnect.